app/src/org/commcare/provider/InstanceProvider.java

```diff
@@ -520,7 +520,8 @@ private void linkToSessionFormRecord(Uri instanceUri) {
             return;
         }
 
-        Logger.log(AndroidLogger.TYPE_FORM_ENTRY, "Form Entry Completed");
+        Logger.log(AndroidLogger.TYPE_FORM_ENTRY,
+                String.format("Form Entry Completed for record with id %s", current.getInstanceID()));
```

Comment:
Is the instanceID here _our_ instance Id, or is it the ID of the content provider?